oeis.py

```diff
@@ -120,6 +121,12 @@ def A000290(start: int = 0, limit: int = 20) -> Collection[int]:
     return sequence
 
 
+@oeis
+def A001221(start: int = 0, limit: int = 20) -> Collection[int]:
+    "Number of distinct primes dividing n (also called omega(n))."
+    return [len(primefactors(n)) for n in range(start, limit)]
```

Comment:
```suggestion
    return [len(primefactors(n)) for n in range(start, start + limit)]
```

Les paramètres de `range` c'est `start, end` pas `start, limit`. Pour convertir de l'un à l'autre utilise `range(start, start + limit)`.